### PR TITLE
[9.0][FIX] .travis: Update to latest template

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,42 +1,64 @@
...
+#     Search your packages here:
+#       https://github.com/travis-ci/apt-package-whitelist/blob/master/ubuntu-precise
+#     For wkhtmltopdf, see the env section below
 
 env:
   global:
-  - VERSION="9.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
-  - TRANSIFEX_USER='transbot@odoo-community.org'
-  - secure: "WspsOy3m3Arreqs+f8QAnKOCPbXfpO7TeUJEfJSMehMh8I+bR1KFZM0CUfKxNOT9mrbt94J9VzF3knC2p4P/lUWr24USJ27FgOScnTz/PQTjZfjdXDefGdhCCx3QgIzI/mj751BYp2FMBYXEvjaBSycerWJ+AP+OKVEdf44Lgp0="
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" MAKEPOT="0"
```
Comment (pedrobaeza) [inline .travis.yml]: Incorrect version

File: .travis.yml [hunk 2]
```diff
@@ -1,42 +1,64 @@
...
+#  - TESTS="1.0" ODOO_REPO="OCA/OCB" UNIT_TEST="1"
 
 install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
   - pip install PyPDF2==1.18
   - pip install unidecode
-  - pip install pycountry
+  - pip install pycountry>=16.11.08
```
Comment (pedrobaeza) [inline .travis.yml]: You can take the occasion to move it to requirements.txt

File: .travis.yml [hunk 3]
```diff
@@ -1,42 +1,64 @@
...
-    source:
-      - pov-wkhtmltopdf
+#    sources:
+#    Search your sources alias here:
+#      https://github.com/travis-ci/apt-source-whitelist/blob/master/ubuntu.json
     packages:
       - expect-dev  # provides unbuffer utility
-      - python-lxml # because pip installation is slow
-      - wkhtmltopdf
+      - python-lxml  # because pip installation is slow
```
Comment (pedrobaeza) [inline .travis.yml:20]: Is this still needed?

File: .travis.yml (hunk 1 above)
Comment (rousseldenis) [reply .travis.yml]: copy/paste :smile:

File: .travis.yml (hunk 3 above)
Comment (rousseldenis) [reply .travis.yml:20]: I let the original one

**Comment by rousseldenis** [comment]
@pedrobaeza Done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**